trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.length > 10 ? "#{@nameable.correct_name[0, 10]}..." : @nameable.correct_name
```

Comment:
- [ ] **[OPTIONAL]** There is no need for the three dots. The string should be 10 characters long. Adding three dots makes it 13 characters, longer than even the initial input, in this case.

  ![image](https://github.com/Ayokunnumi1/school-library/assets/36542986/a1c34988-442d-4e06-a164-95ed3f2f255e)

Reply:
Thanks for your corrections, I have resolved them